templates.rst

```diff
@@ -191,6 +191,7 @@ Consider the following routing configuration:
         namespace App\Controller;
 
         // ...
+        use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
```

Comment:
There is a flow in this page, and we should either shorten each start of files to show only what's relevant (what we usually prefer), or remove the three dots comment.

Reply:
The use statement is relevant in the sense that it is directly included in the given code example, pending your approval.